download_url

```diff
@@ -72,6 +74,12 @@ while test $# -gt 0; do
       sslvalidation="$2"
       shift
     ;;
+    *-download-manifest)
+      download_manifest=$2
+      shift
+      url_list=`cat $download_manifest`
```

Comment:
that's rather nasty. wget has a -i parameter that would be useful here

Reply:
Thanks for the hint